lib/config/config_test.go

```diff
@@ -737,7 +737,7 @@ func TestV14ListenAddressesMigration(t *testing.T) {
 		// config to start with...
 		{
 			{"tcp://0.0.0.0:22000"}, // old listen addrs
-			{""},                    // old relay addrs
+			{""}, // old relay addrs
```

Comment:
This change needs to go, then this PR looks good to me.

Reply:
This was gofmt isn't that the format used in this repo?

Reply:
It is. I think the comment alignment is a recent change - are you by chance on an older go(fmt) version?

Reply:
Yes probably, just using standard ubuntu packages